src/types/tests/search.test-d.ts

```diff
@@ -0,0 +1,386 @@
+				display_name: "All",
+				count: 454,
+				data: {},
+				children: [
+					{
+						group_id: "310-000-000-000",
+						display_name: "Trailer",
+						count: 62,
+						data: {},
+						children: [],
+						parents: [
+							{
+								display_name: "All",
+								group_id: "All",
+							},
+						],
+					},
+				],
+				parents: [],
+			},
+		],
+		results: [
+			{
+				matched_terms: ["red"],
+				data: {
+					id: "PA0009015918",
+					url: "https://www.princessauto.com/en/27-ton-212cc-gas-powered-log-splitter/product/PA0009015918",
+					keywords: ["Red Rock"],
+					image_url:
+						"https://www.princessauto.com/ccstore/v1/images/?source=/file/products/9015918_A0CG_00_01.jpg",
+					group_ids: ["220039", "420-010-015-010"],
+					deactivated: false,
+					description:
+						"Use this log splitter to split logs up to 26 in. long. It features horizontal and vertical operation, low-oil shutdown, auto cylinder return, 14-second cycle time and powder-coated steel construction.",
+					paProductStatus: "A",
+					paProductClearance: false,
+					listPrice: 1799.99,
+					SaleDates: "Oct 10 2022 - Oct 23 2022",
+					salePrice: 1399.99,
+					variation_id: "9015918",
+					selectorImage: "9015918_A0CG_00_01.jpg",
+				},
+				value: "27 Ton 212cc Gas-Powered Log Splitter",
+				is_slotted: false,
+				labels: {},
+				variations: [
+					{
+						data: {
+							listPrice: 1799.99,
+							SaleDates: "Oct 10 2022 - Oct 23 2022",
+							salePrice: 1399.99,
+							deactivated: false,
+							variation_id: "9015918",
+							selectorImage: "9015918_A0CG_00_01.jpg",
+						},
+						value: "27 Ton 212cc Gas-Powered Log Splitter",
+					},
+				],
+			},
+		],
+		sort_options: [
+			{
+				sort_by: "relevance",
+				display_name: "Relevance",
+				sort_order: "descending",
+				status: "selected",
+			},
+		],
+		refined_content: [],
+		total_num_results: 454,
+		features: [
+			{
+				feature_name: "auto_generated_refined_query_rules",
+				display_name: "Affinity Engine",
+				enabled: true,
+				variant: {
+					name: "default_rules",
+					display_name: "Default weights",
+				},
+			},
+		],
+	},
+	result_id: "dda3208c-0f0c-412e-92d9-7abf1d21e228",
+	request: {
+		page: 1,
+		num_results_per_page: 50,
+		section: "Products",
+		blacklist_rules: true,
+		term: "red",
+		fmt_options: {
+			groups_start: "current",
+			groups_max_depth: 1,
+		},
+		sort_by: "relevance",
+		sort_order: "descending",
+		features: {
+			query_items: true,
+			auto_generated_refined_query_rules: true,
+			manual_searchandizing: true,
+			personalization: true,
+			filter_items: true,
+		},
+		feature_variants: {
+			query_items: "query_items_ctr_l2r_ctr_ss",
+			auto_generated_refined_query_rules: "default_rules",
+			manual_searchandizing: null,
+			personalization: "default_personalization",
+			filter_items: "filter_items_w_atcs_and_purchases",
+		},
+		searchandized_items: {},
+	},
+});
+
+// Example response from https://bonobos.com/search?term=red
```

Comment:
I think we should probably remove references to customers here. So changing up the image links and what not.  Same for above.